projects/storefrontlib/src/feature-toggles.ts

```diff
@@ -7,5 +7,18 @@ export interface FeatureToggles {
      */
     level?: string;
     consignmentTracking?: boolean;
+
+    /**
+     * With 3.0 we introduced an unified configuration that emits new configuration
+     * every time lazy loaded module is instantiated. To ease transition to this new
+     * mechanism we are also publishing those changes to global Config token under
+     * the hood which can cause issues in some implementations.
+     *
+     *This option allows to disable this mechanism, i.e.:
+     *  - configuration won't change after app will be bootstrapped
+     *  - if any service would like to take into account configuration from lazy loaded
+     *    modules, it will have to use ConfigurationService.unifiedConfig$
+     */
```

Comment:
```suggestion
     * The version 3.0 introduced an unified configuration that emits new configuration
     * every time lazy loaded module is instantiated. To ease transition to this new
     * mechanism, the changes to global Config token are also published under
     * the hood which can cause issues in some implementations.
     *
     * This option allows to disable this mechanism, i.e.:
     *  - configuration won't change after app will be bootstrapped
     *  - if any service would like to take into account configuration from lazy loaded
     *    modules, it will have to use ConfigurationService.unifiedConfig$
     */
```

Reply:
Thx, fixed!